AppImage-package/bundle.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+########################################################################
+# Package the binaries built on Travis-CI as an AppImage
+# By Simon Peter 2016
+# For more information, see http://appimage.org/
+########################################################################
+sudo apt-get -y install coreutils binutils python-tk
+export ARCH=$(arch)
+APP=Scribus
+LOWERAPP=${APP,,}
+GIT_REV=$(git rev-parse --short HEAD)
+echo $GIT_REV
+sudo make install DESTDIR=/$APP/$APP.AppDir
+sudo chown -R $USER /$APP
```

Comment:
I don’t have any experience with AppImage, and I also prefer to not use sudo before this is merged.